app/components/candidate_interface/deadline_banner_component.rb

```diff
@@ -18,30 +26,34 @@ def render?
 private
 
   def show_apply_1_deadline_banner?
-    apply_1? &&
-      CycleTimetable.show_apply_1_deadline_banner? &&
-      FeatureFlag.active?(:deadline_notices)
+    apply_1? && CycleTimetable.show_apply_1_deadline_banner?
```

Comment:
Both the banner checks should check the status of the application — we don't want to show for submitted apps.